costumedb-frontend/src/app/search.service.ts

```diff
@@ -8,20 +8,30 @@ import { CostumeRequestObject } from './models/costume-request.model';
   providedIn: 'root'
 })
 export class SearchService {
+<<<<<<< HEAD
+<<<<<<< HEAD
   // apiURL = "https://run.mocky.io/v3/4dc26555-f40e-46ba-82a4-1b733735725e"
   apiURL = "https://run.mocky.io/v3/ef87c623-8407-416d-819d-0bf6829d3d21"; // this one is blank for testing empty response
+=======
+  apiURL = "http://localhost:5000/groupsearch"
+=======
+  apiURL1 = "http://localhost:5000/groupsearch"
+>>>>>>> 0387e95 (add endpoints for text and tag search)
+  // apiURL = "https://run.mocky.io/v3/ef87c623-8407-416d-819d-0bf6829d3d21"; // this one is blank for testing empty response
+>>>>>>> 0dba4f0 (create endpoint for groupsearch, add flask_cors)
```

Comment:
Remains of a merge conflict here, remove the `<<<<<<< HEAD`, `=======`, `>>>>>>> 0387e95` etc